ovirtlago/testlib.py

```diff
@@ -209,7 +209,7 @@ def _instance_of_any(obj, cls_list):
     return any(True for cls in cls_list if isinstance(obj, cls))
 
 
-def assert_equals_within(func, value, timeout, allowed_exceptions=None):
+def assert_equals_within(func, value, timeout, allowed_exceptions=None, initial_wait=10):
```

Comment:
The initial wait takes place only after the first run of the function. So if it fails miserably right away, we are safe. It adds 7 seconds to what we've had before, that's all.
Note that anyway failures are fatal, so it doesn't matter usually.